### PR TITLE
Chore/config prettier con eslint

### PR DESCRIPTION
**Pull request by mcljs**
Configuración de Prettier con Eslint

### CODE REVIEW

**Comment by darthzyklus** [comment]
@mcljs please resolve the conflicts to continue

**Review by darthzyklus — CHANGES_REQUESTED** [review]
the PR needs to resolve the conflicts to continue and merge into develop.

**Review by darthzyklus — APPROVED** [review]

**Comment by Hmerin** [comment]
@mcljs I think the `.vscode` directory shouldn’t be committed or just preserve the following lines:

``` js
{
  "editor.formatOnSave": false,
  "editor.codeActionsOnSave": {
      "source.fixAll.eslint": true
  }
}
```

That config file makes my vscode looks this way:

Screen Shot 2021-03-20 at 10 09 35 PM

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "window.zoomLevel": 5,
```
Comment (Hmerin) [inline .vscode/settings.json:2]: This line modifies the other ppl editors, please left the eslint/prettier related configs

**Review by Hmerin — COMMENTED** [review]
@mcljs Please edit the description providing an example of how to test this config or how to trigger the `codeActionsOnSave` I had no luck editing some files expecting this config to make the lint automatically

**Comment by mcljs** [comment]
The configuration and issues with this branch were resolved in PR # 32 . Details which were:

problems in the fontsize .vscode
conflict issues with the develop branch

**Comment by Hmerin** [comment]
PR closed in favor of #32

**Pull request closed without merging**